edx/edx-ora2
# [BD-05][EDUCATOR-5596] Remove tinymcev5 and use the existing one from the platform

During testing, it has been discovered that the TinyMCE v5 used in the Pluggable ORA Submission editor collides with the existing TinyMCE v4 in the platform. This PR removes TinyMCE v5 and uses the version from the platform instead.

**JIRA tickets**: https://openedx.atlassian.net/browse/EDUCATOR-5596

~~**Discussions**:~~

**Dependencies**: None

**Screenshots**:
<img width="938" alt="Screenshot 2021-02-26 at 1 09 32 PM" src="https://user-images.githubusercontent.com/1010244/109268551-6fd8a180-7835-11eb-8d7c-a18e508f9a65.png">

<img width="856" alt="Screenshot 2021-02-26 at 1 11 29 PM" src="https://user-images.githubusercontent.com/1010244/109268565-7535ec00-7835-11eb-8f9c-3f557c09045c.png">


**Sandbox URL**: TBD

**Merge deadline**: None

**Testing instructions**:

1. Pull this PR into your local devstack.
2. Install `edx-ora2`.
3. Create a ORA block in studio.
4. From Edit -> Settings -> Response Editor, select WYSIWYG Editor.
5. Check that TinyMCE Response Editor loads in Studio.
6. Check that ORA Prompt editor works.
7. Publish the block.
8. Check TinyMCE loads in LMS.

**Author notes and concerns**:

TBD

**Reviewers**
